src/codec.cairo

```diff
@@ -29,13 +30,68 @@ pub impl EncodeByteArrayImpl of Encode<@ByteArray> {
         dest.append(self);
     }
 }
+pub impl EncodeTxIn of Encode<TxIn> {
+    fn encode_to(self: TxIn, ref dest: ByteArray) {
+        self.script.encode_to(ref dest);
+        dest.append_word_rev(self.sequence.try_into().unwrap(), 4);
```

Comment:
I think u32 can be casted with just .into()

Anyways, it would be more clean if you implement Encode<u32> and Encode<i64>